### PR TITLE
4486 cross case keyword search

### PR DESCRIPTION
**Pull request by dannysmyda**

### CODE REVIEW

File: KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/multicase/MultiCaseKeywordSearchPanel.java [hunk 1]
```diff
@@ -0,0 +1,907 @@
...
+        casesLabel = new javax.swing.JLabel();
+        resultsLabel = new javax.swing.JLabel();
+        uncheckButton = new javax.swing.JButton();
+        checkButton = new javax.swing.JButton();
+        toolDescriptionScrollPane = new javax.swing.JScrollPane();
+        toolDescriptionTextArea = new javax.swing.JTextArea();
+        resultsScrollPane = new javax.swing.JScrollPane();
+        cancelButton = new javax.swing.JButton();
+        searchProgressBar = new javax.swing.JProgressBar();
+        warningLabel = new javax.swing.JLabel();
```
Comment (wschaeferB) [inline KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/multicase/MultiCaseKeywordSearchPanel.java]: This warningLabel should be made horizontally re-sizable

File: KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/multicase/MultiCaseKeywordSearchPanel.java [hunk 2]
```diff
@@ -0,0 +1,907 @@
...
+     * regenerated by the Form Editor.
+     */
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+    private void initComponents() {
+
+        searchTypeGroup = new javax.swing.ButtonGroup();
+        searchButton = new javax.swing.JButton();
+        substringRadioButton = new javax.swing.JRadioButton();
+        keywordTextField = new javax.swing.JTextField();
```
Comment (wschaeferB) [inline KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/multicase/MultiCaseKeywordSearchPanel.java]: the keywordTextField should be made horizontally resizable

**Comment by dannysmyda** [comment]
This still needs one last fix to it.

**Comment by dannysmyda** [comment]
This PR should be all done now.

**Pull request merged**